src/ontology/cl-edit.owl

```diff
@@ -28903,7 +28903,8 @@ AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_3000001 <https
 AnnotationAssertion(oboInOwl:creation_date obo:CL_3000001 "2019-02-18T19:41:20Z")
 AnnotationAssertion(oboInOwl:id obo:CL_3000001 "CL:3000001")
 AnnotationAssertion(rdfs:label obo:CL_3000001 "Hofbauer cell")
-SubClassOf(obo:CL_3000001 obo:CL_0000235)
+SubClassOf(obo:CL_3000001 obo:CL_0000864)
+SubClassOf(obo:CL_3000001 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0003124))
```

Comment:
Text definition invokes "placenta" and "chorionic villus". Consider aligning.

Reply:
- "an heterogenic" --> "a heterogenic". ("a" precedes | hɛ |)
- "fetal derived" --> "fetal-derived" (compound adjective)
- "chorionic lavae" --> "chorion lavae" (cursory online check shows that the latter is more common)

Reply:
- "lavae --> "laeve" (tracing back to the [reference cited](https://pubmed.ncbi.nlm.nih.gov/30498493/) in the paper, it is "chorion laeve")

Reply:
All changes have been addressed